docs/tutorials/light-up.md

```diff
@@ -0,0 +1,224 @@
+This robot will turn the lights on or off when it detects a person in front of it.
+You can turn it into a night light for reading books, a security robot that alerts you when a person is close by, or a bathroom light that only activates when people enter; the opportunities are endless.
+
+This project is a great place to start if you are new to building robots, and have a smart plug laying around.
+
+{{< gif webm_src="../img/light-up/light-up.webm" mp4_src="../img/light-up/light-up.mp4" alt= "The project working: a person sitting at a desk with a computer and lightbulb set up in front of her. As she leaves the light turns off, and as she enters the frame, the light turns back on." >}}
+
+## Hardware requirements
+
+The following hardware is used in this tutorial:
+
+- Computer with a webcam
+  - This tutorial uses a MacBook Pro but any computer running macOS or 64-bit Linux will work
+- Mobile phone (to download the Kasa Smart app)
+- [Kasa Smart Wi-Fi Plug Mini](https://www.kasasmart.com/us/products/smart-plugs/kasa-smart-wifi-plug-mini)
+- [Kasa Smart Light Bulb](https://www.kasasmart.com/us/products/smart-lighting/kasa-smart-light-bulb-multicolor-kl125)
+- [Table Lamp Base](https://www.amazon.com/gp/product/B08KZNZVY7/ref=ppx_yo_dt_b_asin_title_o02_s00?) or similar
+
+## Software requirements
+
+The following software is used in this tutorial:
+
+- [Python3](https://www.python.org/downloads/)
+- [`viam-server`](/installation/install/)
+- [Viam Python SDK](https://python.viam.dev/)
+- [Project repo on GitHub](https://github.com/viam-labs/devrel-demos/tree/main/Light%20up%20bot)
+
+## Prerequisites
+
+Make sure you have a recent version of Python installed on your computer by running
+
+```bash
+python --version
+```
+
+If the output is Python 3.9.0 or later, you should be fine for this tutorial.
+If not, [download a newer version here](https://www.python.org/downloads/).
+
+The Viam Python SDK (software development kit) lets you control your robot by writing custom scripts in the Python programming language.
+Install the Viam Python SDK by following [these instructions](https://python.viam.dev/).
+
+Now you are ready to make your robot!
+
+## Install `viam-server` and connect to your robot
+
+Go to the [Viam app](https://app.viam.com) and make or log in to your account.
+
+Create a new robot.
+
+Go to the **SETUP** tab of your new robot's page.
+Select the correct **Mode** (Linux or Mac) and (if you're on Linux) **Architecture** for your computer.
+Most Linux desktops and laptops use **X86_64** architecture.
+
+Follow the steps on this tab to install `viam-server` on your computer.
+
+## Configure object detection
+
+On your new robot's page, go to the **CONFIG** tab.
+
+![The CONFIG tab in Builder mode on the Viam app.](../img/light-up/config-tab.png)
+
+On the **CONFIG** tab, create a new component:
+
+- **Name**: `my-camera`
+- **Type**: `camera`
+- **Model**: `webcam`
+
+Click **Create Component** to add the camera.

```

Comment:
```suggestion
On your new robot's page, go to the **CONFIG** tab and create a new component with:

- **Name**: `my-camera`
- **Type**: `camera`
- **Model**: `webcam`

![The CONFIG tab in Builder mode on the Viam app.](../img/light-up/config-tab.png)
```

Reply:
also the naming i chose in the tutorial was camera-mac so all the gifs and recordings i sent as well as the final code refer to that

Reply:
I don't think this makes sense since the screenshot just shows the empty config tab

Reply:
i think its better if we make it as generic as possible but say "i am using a mac so i named mine camera-mac". its really confusing to have different naming in **Name**: `my-camera`, code, and screenshots/videos